src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,8 @@
+NuGet.PackageManagement.IDEExecutionContext.IDEExecutionContext(NuGet.PackageManagement.ICommonOperations commonOperations, NuGet.PackageManagement.IRenderReadMeMarkdownToolWindow renderReadMeMarkdownToolWindow) -> void
+NuGet.PackageManagement.IDEExecutionContext.Render.get -> NuGet.PackageManagement.IRenderReadMeMarkdownToolWindow
+NuGet.PackageManagement.IRenderReadMeMarkdownToolWindow
+NuGet.PackageManagement.IRenderReadMeMarkdownToolWindow.DisplayReadMeMarkdownToolWindowAsync(string filePath) -> System.Threading.Tasks.Task
+abstract NuGet.ProjectManagement.ExecutionContext.RenderMarkDownFile(string fullPath) -> System.Threading.Tasks.Task
 const NuGet.ProjectManagement.ProjectBuildProperties.CLRSupport = "CLRSupport" -> string
+override NuGet.PackageManagement.IDEExecutionContext.RenderMarkDownFile(string fullPath) -> System.Threading.Tasks.Task
+static readonly NuGet.ProjectManagement.Constants.ReadmeFileNameMarkdown -> string
```

Comment:
`NuGet.PackageManagement` is used by `nuget.exe` and some customers in their own apps. While these apps can certainly just not use these APIs, it does feel like these APIs are being added to an assembly where they don't belong.

I didn't look at the PR in detail, so there might be a good reason why they're hard to move to a VS-specific assembly (like NuGet.PackageManagement.VisualStudio),  but in case you haven't tried already, please do.

Some customers expect APIs in our packages that we publish to nuget.org to work for them, and these APIs will only work in our own Visual Studio extension. Therefore, if these APIs ship, they might increase our support workload.